styles/lib/carnival/features/_tables.scss

```diff
@@ -0,0 +1,357 @@
+
+@use 'lib/carnival/colors';
+@use 'lib/carnival/fonts';
+
+@mixin basic-table($border-after-table: colors.$black) {
+    .os-table:not(.os-unstyled-container) {
+      margin-bottom: 0.7rem;
+      display: table;
+      margin-left: auto;
+      margin-right: auto;
+      > table {
+        border-collapse: collapse;
+        width: 100%;
+        margin-bottom: 0;
+        > thead > tr {
+          > th {
+            border-bottom: 0.1rem solid colors.$black;
+            border-top: 0.1rem solid colors.$black;
+            font-family: fonts.$mulish;
+            font-weight: 700;
+            padding: 0.7rem;
+            &[data-valign=top] {
+              vertical-align: top;
+            }
+            &[data-align=left] {
+              text-align: left;
+            }
+            &[data-align=right] {
+              text-align: right;
+            }
+           &[data-align=center] {
+              text-align: center;
+            }
+          }
+        }
+        > tbody > tr > td {
+          &[data-valign=top] {
+              vertical-align: top;
+            }
+          &[data-align=left] {
+              text-align: left;
+            }
+          &[data-align=right] {
+              text-align: right;
+            }
+          &[data-align=center] {
+              text-align: center;
+            }
+        }
+      }
+    }
+    .os-table:not(.os-unstyled-container) {
```

Comment:
this is a lot closer! however it looks like this selector is still a repeat of the one on line 6. i wonder if it can be condensed even more? 🤔

Reply:
Yep you're right. I've condensed this mixin.